net/dns/manager_windows_test.go

```diff
@@ -332,6 +435,37 @@ func getSavedDomainsForRule(base, ruleID string) ([]string, error) {
 	return result, err
 }
 
+func genRandomSubdomains(t *testing.T, n int) []dnsname.FQDN {
```

Comment:
I'm generally a fan of avoiding randomly generated test cases unless it uses a fixed seed or displays the seed because sometimes it results in flakes if certain inputs fail but you can't reproduce what the test did.

Reply:
I don't expect too many problems here given that I'm limiting the length of the generated text and restricting it to the lowercase English alphabet, but I see what you're getting at. I'll log the seed.